src/MongoDB/Monitoring/CommandFailedEvent.stub.php

```diff
@@ -13,6 +13,8 @@ final private function __construct() {}
 
     final public function getCommandName(): string {}
 
+    final public function getDatabaseName(): string {}
```

Comment:
Nothing to change here as all values are exposed through a getter, but we can consider changing all of these methods to readonly properties sometime down the line.